.travis.yml

```diff
@@ -1,23 +1,29 @@
 language: python
 python:
-  - "3.7"
+- "3.7"
 
 install:
-  - pip install .
-  - pip install -r requirements-test.txt
+- pip install .
+- pip install -r requirements-test.txt
 
 script:
-  - coverage run -m pytest
-  - coverage report -m
+- coverage run -m pytest
+- coverage report -m
 
-# Not fully tested as travis isn't working anymore for some reason.
-# TODO: create a token for production PyPI, encrypt it, paste-it below and remove `server` params below.
 deploy:
-  provider: pypi
+- provider: pypi
   username: __token__
-  server: https://test.pypi.org/legacy
+  server: https://test.pypi.org/legacy/
```

Comment:
Why still on test?

Reply:
For tags ending in `-test.n` where `n` is a number. So we can test that if a version builds correctly.